python/private/pypi/dependency_specifier_flag.bzl

```diff
@@ -0,0 +1,84 @@
+load("//python/private:toolchain_types.bzl", "TARGET_TOOLCHAIN_TYPE")
+load(":pep508_evaluate.bzl", "evaluate")
+
+def _impl(ctx):
+    env = {}
+
+    runtime = ctx.toolchains[TARGET_TOOLCHAIN_TYPE].py3_runtime
```

Comment:
I was wondering if we should get it from the toolchain info or the `//python/config_settings:python_version` flag. Probably better from toolchain as it will be more representative, but then we need to fetch the toolchain.